src/php/server_fpm.rs

```diff
@@ -96,26 +100,36 @@ pub(crate) fn start(php_bin: String) -> (PhpServer, Child) {
         .replace("{{ log_level }}", FPM_DEFAULT_LOG_LEVEL)
         .replace("{{ systemd }}", if systemd_support { "" } else { ";" });
 
-    let home = env::var("HOME").unwrap_or(String::from(""));
-
-    let fpm_config_file_path;
-
-    if home != "" {
-        fpm_config_file_path = PathBuf::from(home.as_str())
-            .join(".rymfony")
-            .join("fpm-conf.ini");
-    } else {
-        panic!("Cannot find the \"HOME\" directory in which to write the php-fpm configuration file.");
-    }
+    let rymfony_project_path =  match get_rymfony_project_directory() {
+        Ok(e) => e,
+        _ => panic!("Cannot find the \"HOME\" directory in which to write the php-fpm configuration file.")
+    };
+    let fpm_config_file_path = rymfony_project_path.join("fpm-conf.ini");
 
     if !fpm_config_file_path.exists() {
         let mut fpm_config_file = File::create(&fpm_config_file_path).unwrap();
         fpm_config_file.write_all(config.as_bytes())
              .expect("Could not write to php-fpm config file.");
+    } else {
+        // Read the file and search the port
+        let content = read_to_string(&fpm_config_file_path).unwrap();
+        let re = Regex::new(r"listen[ ]?=[ ]?(?:(?:127\.0\.0\.1|localhost):)?(\d{1,5})").unwrap();
+        let caps = re.captures(content.as_str()).unwrap();
+        let port_txt = caps.get(1).map_or("0", |m| m.as_str());
+        let port_num :u16 = port_txt.parse().unwrap();
+        //Check if the port is available
```

Comment:
No need, the `panic!()` statements already explain what's the goal of this code :)

```suggestion
```